packages/core/src/integrations/mcp-server/types.ts

```diff
@@ -0,0 +1,166 @@
+/**
+ * types for MCP server instrumentation
```

Comment:
Are these vendored in from the mcp package? If so we need to put the mcp library license + sha we grabbed the types from in this file.